HHS/Head-Start-TTADP
# [TTAHUB-2396] New side navigation

## Description of change

Update side navigation to match new mockup

## How to test

Confirm that the navigation starts open, displays pages in the correct order, works as you'd expect, and matches mockup

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2396


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
